catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -118,18 +226,9 @@ public Metacard transform(InputStream input, String id)
         }
     }
 
-    private Metacard transformWithoutExtractors(InputStream input, String id) throws IOException {
-        try (PDDocument pdfDocument = PDDocument.load(input)) {
-            return transformPdf(id, pdfDocument);
-        } catch (InvalidPasswordException e) {
-            LOGGER.warn("Cannot transform encrypted pdf", e);
-            return initializeMetacard(id);
-        }
-    }
-
     private Metacard transformWithExtractors(InputStream input, String id)
             throws IOException, CatalogTransformerException {
-        try (FileBackedOutputStream fbos = new FileBackedOutputStream(1000000)) {
+        try (TemporaryFileBackedOutputStream fbos = new TemporaryFileBackedOutputStream()) {
```

Comment:
This method isn't using your new `inputStreamPDDocumentFunction`. Is that an issue?